compiler/luci/service/src/CircleShapeInferenceRule.cpp

```diff
@@ -492,6 +492,38 @@ loco::NodeShape infer_batchmatmul_shape(const loco::TensorShape &x_shape,
   return loco::NodeShape{output_shape};
 }
 
+loco::NodeShape infer_broadcast_to(const luci::CircleBroadcastTo *node)
+{
+  LOGGER(l);
```

Comment:
where is `l` used?

Reply:
@seanshpark 

`infer_broadcast_to` function is based on the `infer_reshape function`. -> [code](https://github.com/Samsung/ONE/blob/d423975e86813289158c12760a879df90787ebf9/compiler/luci/service/src/CircleShapeInferenceRule.cpp#L1046)
Is it better to remove LOGGER?

Reply:
I think you don't know what is `LOGGER(l)`. Please follow the codes and how it's used.
And you'll get the answer.

Reply:
@seanshpark 
Oh, I knew I'd have to get rid of it because `LOGGER(l)` only declares here and doesn't use it from behind.
Thank you for letting me know. :)